Presentation.Web/app/models/it-system-usage/it-system-usage-validation-details-response-dto.ts

```diff
@@ -0,0 +1,11 @@
+﻿module Kitos.Models.ItSystemUsage {
+
+    /** Contains info about it system usage validity.
+     * @property valid - is true if system usage is valid
```

Comment:
why not add the comment above the property like in all of the other files where comments are used?